pages/admin/form/admin.vue.js

```diff
@@ -0,0 +1,60 @@
+const adminComponent = {
+    data: function() {
```

Comment:
- If you don't want to load or process any data, you can remove this part of the component.

Reply:
Yes, I though that in future we can make some table which will show all data from category, product and tags collection. Is there any harm to leave like this for now?